tests/calico_testcases/example_policies/testcase1/networkpolicy-with-ns-selector-bad-path-test.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: projectcalico.org/v3
+kind: NetworkPolicy
+metadata:
+  name: bad-network-policy
+  namespace: kube-system
+spec:
+  selector: has(app)
+  namespaceSelector: unique-label == 'defaultNameSpace'
+  types:
+  - Ingress
+  - Egress
```

Comment:
Shouldn't this policy be in a scheme file `networkConfig ` with `expectedError`?

Reply:
I've added it as the last `networkConfig ` in `tests/calico_testcases/example_policies/testcase1/testcase1-scheme.yaml`